app/Http/Controllers/UserController.php

```diff
@@ -14,6 +17,20 @@ public function get(User $user)
         return $user;
     }
 
+    public function catchThief(User $user)
+    {
+        if ($user->status != UserStatuses::Playing) {
+            response()->json(['errors' => [
+                'value' => ['Alleen spelers die niet gevangen of in de lobby zijn kunnen gevangen worden.']
+            ]], 422)->throwResponse();
+        }
```

Comment:
Misschien kan er nog validatie toegevoegd worden of de user wel een boef is.

Reply:
De verwachting is eigenlijk dat dat niet nodig is omdat in de app geen mogelijkheid is om een politieagent op te pakken. Het zou dan een beetje nutteloze validatie zijn. Ben je het daarmee eens?

Also welke comment? Ik kan niet vinden waar je het over hebt.

Reply:
> De verwachting is eigenlijk dat dat niet nodig is omdat in de app geen mogelijkheid is om een politieagent op te pakken. Het zou dan een beetje nutteloze validatie zijn. Ben je het daarmee eens?

Het is meer een kwaliteitswaarborging dingetje. Dat er in de app geen mogelijkheid is is een aanname, maar wel een terechte aanname. Ik verwacht ook niet dat het nodig is en vind het ook wel best als je het zo laat.

> Also welke comment? Ik kan niet vinden waar je het over hebt.

Ik bedoelde met "comment" letterlijk mijn opmerking over dit stukje code :p